src/civic/index.js

```diff
@@ -85,6 +84,9 @@ const translateRelevance = (evidenceType, evidenceDirection, clinicalSignificanc
             }
 
             case 'Functional': {
+                if (clinicalSignificance === 'Unaltered Function') {
```

Comment:
because "unaltered function" has been added to the vocabulary we don't need to alter it here

Reply:
I've removed the if statement in the latest commit